README.md

```diff
@@ -41,6 +41,7 @@
 - **Living Style Guide:** React StoryBook \*
 - **JS Utils:** Lodash
 - **State Tree:** Redux + Redux Observable
+- **Requests:** Fetch
```

Comment:
I dropped Rx ajax and replaced it with fetch throughout all epics since it just isn't quite there yet; too many issues with CORS and poor documentation. Fetch works great.